src/main/java/br/com/sbs/cubatech/security/LoginForm.java

```diff
@@ -0,0 +1,22 @@
+package br.com.sbs.cubatech.security;
+
+public class LoginForm {
+
+    private String email;
+    private String password;
```

Comment:
Daria pra ser `final`, os campos!

```suggestion
    private final String email;
    private final String password;
```